src/bootstrap/src/core/build_steps/clippy.rs

```diff
@@ -114,8 +119,23 @@ impl LintConfig {
             warn: merged(&self.warn, &other.warn),
             deny: merged(&self.deny, &other.deny),
             forbid: merged(&self.forbid, &other.forbid),
+            // FIXME: this skip merging clippy.toml,
+            // but maybe there is a better way?
```

Comment:
What does this comment mean?

Reply:
Function named `merge`, but it don't merge 1 field, so i left this as note, better rephrase this?

Reply:
Why don't we want to merge this field?